ansible/roles/dnf_repos/defaults/main.yml

```diff
@@ -0,0 +1,24 @@
+dnf_repos_rocky_ark_prefix: https://ark.stackhpc.com/pulp/content/{{ ansible_distribution | lower }}/9.4
```

Comment:
Could the Rocky version also be set to a fact here? e.g. `ansible_distribution_version`